### DISCUSSION TITLE
Is it possible to run multiple apps using different ports at the same time?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#988] [Q&A] [answered]
### Question

Hi All,

I wanted to ask if it is possible to run multiple apps using different ports at the same time? 

I am using a windows desktop. The first file, I just ran normal from cmd at port 8080.
The second file, I used a different port number inside my ui.run
``` python
ui.run(port = 8081)
```

But this gives me an error saying "WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions"

Do I need to change or adjust any settings or anything? Much appreciated for the help.

Thanks,
Anindya

**Comment by rodja** [comment, marked as answer]
It should work with different ports. We do it on Linux and Mac all the time. Maybe your Windows system has a firewall running? Can you try starting only one app and using the port 8081? What happens if you are trying 5000, 5001 or something different?

**Reply by Anindya088** [reply]
Hi @rodja , thanks so much. Your advice worked perfectly 😄

I tried 2 files with 5000 and 5001 and both worked. Not sure what the issue happened with 8081. But since it works with 5000 and onwards, so I'm good.

Thanks so much for the help.

Regards,
Anindya
